test/cmd/images.sh

```diff
@@ -72,7 +72,7 @@ os::cmd::expect_success_and_text 'oc describe istag/mysql:latest' 'Image Created
 os::cmd::expect_success_and_text 'oc describe istag/mysql:latest' 'Image Name:'
 name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
 imagename="isimage/mysql@${name:0:15}"
-os::cmd::expect_success 'oc describe "${imagename}"'
+os::cmd::expect_success "oc describe ${imagename}"
```

Comment:
do variables no longer get expanded in the expect functions, or was this just for consistency?